app/res/xml/provider_paths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <external-files-path name="external_files" path="temp/external/"/>
```

Comment:
@amstone326 

need to add a new path here to the full filesystem. external_files is an arbitrary semantic ID. temp/external is a relative path. I don't